Formula/lilv.rb

```diff
@@ -20,14 +21,30 @@ class Lilv < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python@3.10" => [:build, :test]
   depends_on "lv2"
   depends_on "serd"
   depends_on "sord"
   depends_on "sratom"
 
   def install
-    system "./waf", "configure", "--prefix=#{prefix}"
-    system "./waf"
-    system "./waf", "install"
+    system "python3", "./waf", "configure", "--prefix=#{prefix}"
+    system "python3", "./waf"
+    system "python3", "./waf", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <lilv/lilv.h>
+
+      int main(void) {
+        LilvWorld* const world = lilv_world_new();
+        lilv_world_free(world);
+      }
+    EOS
+    system ENV.cc, "test.c", "-I#{include}/lilv-0", "-L#{lib}", "-llilv-0", "-o", "test"
+    system "./test"
+
+    system "python3", "-c", "import lilv"
```

Comment:
Will need the full path as I think the dependency `python@3.10` is not in `PATH` during test

Reply:
Interesting, didn't know that (I figured it would be available, even if keg-only, since it is during build).

Reply:
Yes, there `test` environment tries to mimic the user environment more closely, so keg-only applies there.